zebrad/tests/acceptance.rs

```diff
@@ -351,3 +351,61 @@ fn valid_generated_config(command: &str, expected_output: &str) -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn listener_address_test() -> Result<()> {
+    listener_address(
+        "localhost:8233",
```

Comment:
Avoid conflicts with the generate tests, and any other local zcashd or zebrad instances.
```suggestion
        "localhost:0",
```